src/Unicorn/UnicornLibrary.class.st

```diff
@@ -10,10 +10,26 @@ Class {
 	#category : #Unicorn
 }
 
+{ #category : #accessing }
+UnicornLibrary >> knownMacPaths [
+	"Answer a <Collection> of <String> each one representing known libLLVM locations (not necessarily present in the current system"
+	
+	^ { 
+		'/opt/homebrew/lib/' .
+		'/usr/local/lib/' .
```

Comment:
Because if one migrates from Intel Macs to ARM then /usr/local/lib would prioritize the Intel version and it won't work on ARM. But if you are in Intel Mac then homebrew directory would be empty or not used.

Actually /opt/homebrew could be any path but the idea is to have at least one location where we can set the (pharo version of the library)[https://github.com/pharo-project/pharo-unicorn/tree/unicorn2]